data/transactions/logic/eval.go

```diff
@@ -1884,62 +2023,58 @@ func TxnFieldToTealValue(txn *transactions.Transaction, groupIndex int, field Tx
 	if groupIndex < 0 {
 		return basics.TealValue{}, fmt.Errorf("negative groupIndex %d", groupIndex)
 	}
-	cx := EvalContext{EvalParams: EvalParams{GroupIndex: uint64(groupIndex)}}
+	cx := EvalContext{
+		GroupIndex: groupIndex,
+		Txn:        &transactions.SignedTxnWithAD{SignedTxn: transactions.SignedTxn{Txn: *txn}},
+	}
```

Comment:
I think you're right. Since it only cares about those values for "effects" fields, I think we're ok putting in a blank cx until the debugger wants those values.